skrobot/coordinates/base.py

```diff
@@ -983,9 +983,11 @@ def __init__(self, parent=None, *args, **kwargs):
                                         rot=self.rotation,
                                         hook=self.update)
 
-        if parent is not None:
-            parent.assoc(self)
         self.parent = parent
+        if parent is not None:
```

Comment:
Could you explain that you change the `self.parent = parent` order?

Reply:
In the initial commit https://github.com/iory/scikit-robot/pull/162/commits/13ef4d3ff16350af4768bd484639fea931dff420 , I replaced the order. Then in the last commit, https://github.com/iory/scikit-robot/pull/162/commits/5c505922cbc457f8e89640ca056095658789fd02 I fixed the order to the original one, that is first `self.parent = parent` then `parent.assoc(self)`.